deploy/helm/charts/values.yaml

```diff
@@ -63,6 +63,8 @@ ndmOperator:
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
     tag: 0.9.1
+  podLabels:
+  #  name: openebs-ndm-operator
```

Comment:
Why was this label added?

Reply:
also should it be provided with empty value like below: `{}`

Reply:
I have propagated the name label